### DISCUSSION TITLE
Não faz upload de arquivos: retorna: "mkdir(): Permission denied"

### DISCUSSION THREAD

**Discussion by spaycker** [RamonSilva20/mapos#2534] [General]
pessoal estou tentando fazer uploads de imagens e estou tendo dificuldades, retorna o erro: mkdir(): Permission denied

estou usando o ubuntu server. fiz uma instalação limpa do sistema, e instalei apenas o MapOS.

parece que o erro é causado pelo xampp, mas não tenho certeza, não alterei nenhuma permissão de arquivos ou pastas, fiz a instalação como super usuario.

**Comment by Eletric-Vortex** [comment]
Mande capturas de tela do erro, mas muito provável que seja só dar um `sudo chmod -R 777 /opt/lampp/htdocs` que resolve. Mas isso é uma solução temporária e insegura.

**Reply by spaycker** [reply]
então eu fiz isso e resolveu, mas estou rodando offiline, para colocar online seria complicado dar esse nivel de autorização.
